src/contrast.rs

```diff
@@ -544,6 +571,15 @@ mod tests {
         let actual = threshold(&original, 125u8, ThresholdType::Binary);
         assert_pixels_eq!(expected, actual);
     }
+
+    #[test]
+    fn test_scale_linear() {
+        let input = gray_image!(1u8, 2, 3, 4, 5, 6, 7, 8, 9, 10, 100, 255);
+
+        let expected = gray_image!(10u8, 10, 10, 11, 11, 12, 12, 13, 13, 13, 52, 120);
+
+        assert_eq!(scale_linear(&input, 1, 255, 10, 120), expected);
```

Comment:
```suggestion
        assert_pixels_eq!(scale_linear(&input, 1, 255, 10, 120), expected);
```

This gives a more useful diff if the actual image does not match the expected image.